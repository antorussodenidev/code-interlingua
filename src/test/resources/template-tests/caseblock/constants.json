{
  "type": "int",
  "cases": [
    {
      "value": 1,
      "stmts": [
        { "nodeType": "Statement", "code": "System.out.println(\"one\");" },
        { "nodeType": "Statement", "code": "System.out.flush();" }
      ]
    },
    {
      "value": 2,
      "stmts": [
        { "nodeType": "Statement", "code": "System.out.println(\"two!\");" }
      ]
    }
  ],
  "defaultCase": null
}
