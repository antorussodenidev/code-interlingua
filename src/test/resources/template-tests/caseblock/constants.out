package foo;

class C {
  C(int x) {
    switch (x) {
    case 1:
      System.out.println("one");
      System.out.flush();
      break;
    case 2:
      System.out.println("two!");
      break;
    }
  }

}
